app/js/components/tabs/tabcomponents/drugOrderComponent.jsx

```diff
@@ -3,7 +3,156 @@ import React from 'react';
 const DrugOrderComponent = React.createClass({
     render: function(){
         return (
-            <h1>The DrugOrder Component</h1>
+            <div>
+                <h3>Patients taking specific drugs</h3>
+                <form className="form-horizontal">
+                    <div className="form-group">
+                        <label htmlFor="drug" className="col-sm-2 control-label">Drug(s)</label>
+                        <div className="col-sm-6">
+                            <select className="form-control" multiple="multiple" id="drug" name="drug">
+                                <option value="1">Drug 1</option>
+                                <option value="2">Drug 2</option>
+                                <option value="3">Drug 3</option>
```

Comment:
That's going to be focus of the next sprint. We just want to have a functional UI that the result of the REST calls can be displayed in